wps-common/src/main/resources/config/provenance_template_gridded.ftl

```diff
@@ -79,9 +79,9 @@
 
     <prov:entity prov:id="outputAggregationSettings">
         <#if settingsPath?has_content>
-        <prov:location>https://github.com/aodn/geoserver-config/tree/production/${settingsPath}</prov:location>
+        <prov:location>${settingsPath}</prov:location>
         <#else>
-        <prov:location>https://github.com/aodn/geoserver-config/tree/production/wps/gogoduck.xml</prov:location>
+        <prov:location>https://s3-ap-southeast-2.amazonaws.com/aws-wps-dev-testing/config/templates.xml</prov:location>
```

Comment:
Better to link by default to the github file rather than a test s3 bucket: https://raw.githubusercontent.com/aodn/aws-wps/master/wps-common/src/main/resources/config/templates.xml